src/main/kotlin/calculator/view/OutputView.kt

```diff
@@ -0,0 +1,12 @@
+package calculator.view
+
+class OutputView {
+
+    fun printResult(result: Int) {
+        println(RESULT_MESSAGE + result)
+    }
```

Comment:
tmi이긴 한데 이렇게 단일표현식의 사용도 가능합니다!
```suggestion
    fun printResult(result: Int) = println(RESULT_MESSAGE + result)
```

Reply:
더 깔끔해지는 것 같네요!
이번 미션에선느 참고하겠습니다 🫡